mesonbuild/utils/universal.py

```diff
@@ -1929,7 +1933,8 @@ def update(self, progress: int) -> None:
             self.print_dot()
 
     def close(self) -> None:
-        print('')
+        if not self.disable:
+            print('')
```

Comment:
I know it was already like that, but why the empty string?
```suggestion
            print()
```

Reply:
does not make much difference, but why not. Applied.